client/signup/config/steps-pure.js

```diff
@@ -707,6 +707,10 @@ export function generateSteps( {
 
 		design: {
 			stepName: 'design-picker',
+			apiRequestFunction:
+				currentPage && currentPage.toString().match( /\/start\/setup-site/ )
+					? setThemeOnSite
+					: null,
```

Comment:
(I haven't investigated how this is used yet) but It looks to me like this should probably default to `noop` rather than `null`

Reply:
Actually I don't think using noop is necessary as apiRequestFunction is defined as nullable in `client/lib/signup/flow-controller.ts` 👍